routes/training_documents.rb

```diff
@@ -0,0 +1,33 @@
+require 'net/http'
+
+get '/training-documents/?' do
+	@breadcrumbs << {:text => 'Training Documents'}
+	erb :training_documents, :layout => :fixed
+end
+
+get '/training-documents/sop/?' do
+	@breadcrumbs << {:text => 'Training Documents', :href => '\training-documents'}
```

Comment:
backslash?

Reply:
Fixed. Didn't seem to matter but forward slash is what it should be.